package.json

```diff
@@ -69,7 +69,9 @@
     "wp_org_slug": "woocommerce-square"
   },
   "dependencies": {
-    "@wordpress/data": "^8.1.0",
+    "@woocommerce/product-editor": "^1.4.0",
+    "@wordpress/data": "^7.6.0",
```

Comment:
Looks like we downgraded this dependency. Curious if there's a reason why we needed to do that?

Reply:
There's a React peer dependency conflict after upgrading wordpress/scripts. This is why wordpress/data has been downgraded.

I'll fix all of that in #110 by using [Woo Dependency Extraction](https://www.npmjs.com/package/@woocommerce/dependency-extraction-webpack-plugin).